Guides.md

```diff
@@ -43,7 +53,8 @@ On OS X you will need sshfs installed. The easiest way to install sshfs is to in
 After that you can do:
 
 	$ mkdir CYD-remote
-	$ sshfs <liu-id>@ssh.cyd.liu.se:/cyd/home/<liu-id> CYD-remote
+	$ sshfs <liu-id>@ssh.cyd.liu.se: CYD-remote
```

Comment:
Per default hamnar man i home om man bara anger server: (så de hamnar på samma plats).

> 5 juni 2016 kl. 09:03 skrev Rovanion Luckey <notifications@github.com>:
> 
> In Guides.md:
> 
> > @@ -43,7 +53,8 @@ On OS X you will need sshfs installed. The easiest way to install sshfs is to in
> >  After that you can do:
> >  
> >  	$ mkdir CYD-remote
> > -	$ sshfs <liu-id>@ssh.cyd.liu.se:/cyd/home/<liu-id> CYD-remote
> > +	$ sshfs <liu-id>@ssh.cyd.liu.se: CYD-remote
> Jag förstår. Men då är du sysadmin också. Jag vet att första gången jag blev dirigerad att dumpa över en fil på en webbserver och kom till roten av ett Linux-fs så blev det inget dumpande av, jag hade ingen aning om var jag kunde lägga den.
> 
> Så vi kan resonera som så att dom som vet bättre själva kommer att kunna korta ner den om dom så vill, de andra är nog tacksamma för vägvisningen.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>